docs/migration/2211_35/migration.md

```diff
@@ -1,12 +1,14 @@
-# (EARLY NOTES) Migrating a custom app to use Spartacus with Angular v18
+# Migrating a custom app to use Spartacus 2211.35 with Angular 19
 
 Before upgrading Spartacus to the new version with Angular 18, you need to first:
 
+- **If you've upgraded in the past from Spartacus 6.8 to 2211.19, please first follow the guide**: [Modernize Spartacus apps migrated from 6.8 to 2211.19](./modernize-apps-migrated-from-6.8-to-2211.19.md) and only then please get back here. But apps created initially with Spartacus 2211.19+ should skip this step.
 - upgrade Spartacus to version 2211.32.1
 - install Node 22 version
 - upgrade `@types/node` to version 22
```

Comment:
Looks like `@types/node` is added OOTB only to SSR apps. We can change it to e.g.
```
- if using SSR, upgrade `@types/node` to version 22
```
or to be more generic
```
- if you have dependency `@types/node` in your project, upgrade it to version 22
```

Reply:
```suggestion
- if your project uses SSR (Server-Side Rendering), please upgrade `@types/node` to version 22
```